libs/java/client_common/src/main/java/com/yahoo/athenz/common/messaging/DomainChangeMessage.java

```diff
@@ -23,12 +23,14 @@
 public class DomainChangeMessage {
     
     // Represent the changed object type 
-    enum ObjectType {
+    public enum ObjectType {
         DOMAIN, 
-        ROLE, 
+        ROLE,
+        GROUP,
         POLICY, 
-        SERVICE, 
-        ENTITY
+        SERVICE,
+        ENTITY,
+        TEMPLATE
```

Comment:
Template is also not an object type so I don't expect to notify on templates. Instead when applying templates, we'll be updating roles/policies/services/groups.